pyproject.toml

```diff
@@ -41,3 +41,8 @@ pylint = "^2.14"
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.pytest.ini_options]
+markers = [
+  "integtest: Integrationtest use 'pytest -m integtest'"
```

Comment:
does this work with the spaces here?

Reply:
Copied it from documentation and I didn't see any warnings in CI, so I guess it works.